internal/storage/download.go

```diff
@@ -47,7 +51,7 @@ func Download(svc Service, mux goahttp.Muxer, dec func(r *http.Request) goahttp.
 		}
 		defer reader.Close()
 
-		filename := fmt.Sprintf("enduro-%s.7z", pkg.AipID)
+		filename := fmt.Sprintf("%s-%s.7z", strings.Split(filepath.Base(pkg.Name), ".")[0], pkg.AipID)
```

Comment:
strings.Split(filepath.Base(pkg.Name), ".")[0]

Could you create a `fsutil.FilenameWithoutExt` function in the fsutil package?

Reply:
yes, I will take your edited function and use that in fsutil